Index: jessie-stage2/rootfs/etc/dhcpcd.conf
===================================================================
--- jessie-stage2.orig/rootfs/etc/dhcpcd.conf
+++ jessie-stage2/rootfs/etc/dhcpcd.conf
@@ -23,7 +23,7 @@ option rapid_commit
 
 # A list of options to request from the DHCP server.
 option domain_name_servers, domain_name, domain_search, host_name
-option classless_static_routes
+#option classless_static_routes
 # Most distributions have NTP support.
 option ntp_servers
 # Respect the network MTU.
